app/src/components/HomePage/DownloadAndShareModals/UnpublishedView.tsx

```diff
@@ -0,0 +1,24 @@
+import type {TFunction} from "i18next";
+import {Checkbox, ModalBody, Text} from "@chakra-ui/react";
```

Comment:
Can you run Prettier for code formatting on this file and others? Helps prevent conflicts with other changes when the indentation etc. is consistent.